fluent-react/test/localized_valid_test.js

```diff
@@ -45,13 +45,11 @@ suite('Localized - validation', function() {
   });
 
   test('without a child', function() {
-    function render() {
-      shallow(
-        <Localized />,
-        { context: { l10n: new ReactLocalization([]) } }
-      );
-    }
-    assert.throws(render, /a single React element child/);
+    const wrapper = shallow(
+      <Localized />,
+      { context: { l10n: new ReactLocalization([]) } }
+    );
+    assert.equal(wrapper.length, 1);
```

Comment:
Please bring this test back and add a new one in `localized_render_test.js` which tests `<Localized>` with a string-typed child.

Reply:
The test is still there -- I've just changed it not to check for an exception as it doesn't throw one anymore.

I've added a few test cases for fallback string children (with/without message) and empty children (with/without message) as suggested.